analyzer/src/funTest/assets/projects/synthetic/gradle-all-dependencies-expected-result-with-curations.yml

```diff
@@ -676,6 +537,5 @@ analyzer:
           - "curated license a"
           - "curated license b"
           comment: "Declared license in pom.xml is wrong."
-    errors: {}
```

Comment:
Only semi-related, but I'm wondering: Were we actually ever populating this `errors` map? Even for files like this one where we actually do have errors in the tree, this is empty. Also a quick search through the code indicates we never write to this field. If so, we should probably remove it from the `AnalyzerResult` class in a preceeding commit.

Reply:
Ok, I just tried to make that more clear in https://github.com/heremaps/oss-review-toolkit/pull/946.